tests/test.yml

```diff
@@ -2,5 +2,6 @@
 - hosts: localhost
   remote_user: root
   roles:
-    - role: ansible-role-virtualenvwrapper
-      virtualenvwrapper_shell_rc_file: /home/travis/.bashrc
\ No newline at end of file
+    # Current parameterized role, look at '../defaults/main.yml' to see all variables
+    - role: ../..
+      virtualenvwrapper_shell_rc_file: /home/travis/.bashrc
```

Comment:
@marconius can you also check all the other meta data? like if the minimum version of ansible is still 1.5 for instance